### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by gamperizza**
Updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Backend-registry
+Backend for Register of Clinical Practice projects of St. Petersburg State University.
```
Comment (MicAPic) [inline README.md:2]: Change to 
> Backend of the SPbU Project Registry web app. Created as part of the in-university internship.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Launch
+1. Download [Docker](https://docs.docker.com/engine/install/)
+2. Open src -> main -> docker and run:
```
Comment (MicAPic) [inline README.md:6]: Change to `/src/main/docker/`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  docker-compose up
+  ```
+  
+## Dev notes
+1. Download [Java17](https://www.oracle.com/cis/java/technologies/downloads/) and [Maven](https://maven.apache.org/download.cgi)
+2. In resources -> application properties you can find credentials to deploy the database. 
+
+
+## Database 
+Information and description about database is located and in *DataBase* folder.
```
Comment (MicAPic) [inline README.md:17]: Change to 
>The project information and description can be found in the `/database/` folder

File: README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Software testing
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/20640621-98eced96-51b4-42e0-9eab-d6b96913f4dd?action=collection%2Ffork&source=rip_markdown&collection-url=entityId%3D20640621-98eced96-51b4-42e0-9eab-d6b96913f4dd%26entityType%3Dcollection%26workspaceId%3D2b29e6e1-84ae-4a9a-889b-8218192e6351)
+
+## Asana
+We used Asana for task management throughout the duration of the project. All information is shown in **Project managment** folder.
```
Comment (MicAPic) [inline README.md:24]: Change to `\project_management\`. Please rename the corresponding folder in your 2nd branch

**Review by MicAPic — CHANGES_REQUESTED** [review]
Requested some minor stylistic changes

**Comment by MicAPic** [comment]
Also wait for PR #30 to get approved first, please

**Review by MicAPic — APPROVED** [review]
Everything's good now

**Review by L1mple — APPROVED** [review]
Great job!

**Pull request merged**